tests/features/common/kie-kieserver-common.feature

```diff
@@ -458,3 +458,8 @@ Feature: Kie Server common features
       | MQ_SERVICE_PREFIX_MAPPING | AMQPREFIX |
     Then container log should contain Configuring external JMS integration, removing /opt/eap/standalone/deployments/ROOT.war/META-INF/kie-server-jms.xml
      And file /opt/eap/standalone/deployments/ROOT.war/META-INF/kie-server-jms.xml should not exist
+
+  Scenario: RHDM-1096 - Check that optaplanner thread pool queue size property is not set without env.
```

Comment:
would you mind to add a scenario where you actually set this value as well

Reply:
Sure! done.